source/class/cv/TemplateEngine.js

```diff
@@ -338,6 +338,9 @@ qx.Class.define('cv.TemplateEngine', {
           }
         });
         loader.addListenerOnce('stylesLoaded', this.generateManifest, this);
+        loader.addListenerOnce('stylesAndScriptsLoaded', () => {
+          cv.ui.structure.pure.layout.ResizeHandler.invalidateScreensize();
+        }, this);
```

Comment:
This is only needed for pure-structure and therefore should be done in its controller `cv.ui.structure.pure.Controller`. If you use tile-structure this code would produce an error because the ResizeHandler is not even loaded.